bot.py

```
@@ -307,11 +332,12 @@ async def on_message(message):
         elif is_sum_hr_command:
             await handle_sum_hr_command(message, bot.user)
     except Exception as e:
-        logger.error(f"Error processing command in on_message: {e}", exc_info=True)
+        log_error_with_context(e, f"Processing command in on_message", ErrorSeverity.MEDIUM)
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)